src/ex4/dist/clients/item_client.js

```diff
@@ -1 +1,48 @@
-// Create an ItemClient class here. This is what makes requests to your express server (your own custom API!)
\ No newline at end of file
+// Create an ItemClient class here. This is what makes requests to your express server (your own custom API!)
+
+const getAllTasks = async () => {
```

Comment:
the ItemClient needs to be a class according to the requirements, fix it by the next submission